packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -160,6 +158,7 @@ $border: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
     color: var(--onyx-color-text-icons-neutral-intense);
     text-align: left;
     contain: paint;
+    width: 100%;
```

Comment:
would you like to maybe restrict the width of the storybook examples? ^^ those are huuuge

![image](https://github.com/SchwarzIT/onyx/assets/151019977/7dce74b9-abbc-4eaf-8b22-ac1255c0ae3b)